enclave-tls/samples/enclave-tls-client/client.c

```diff
@@ -217,8 +268,8 @@ int main(int argc, char **argv)
 		case 'p':
 			port = atoi(optarg);
 			break;
-		case 'P':
-			debug_enclave = false;
+		case 'D':
+			debug_enclave = true;
```

Comment:
Why changing to --debug-enclave?

Reply:
lowercase and uppercase P is not easy to be distinguished.  So by debug-enclave it can represent enclave attribute "product or debug"

Reply:
The foundational problem is that the default enclave should be product rather than debug. This is true for sgx2 platform with FLC enabled by default.

Reply:
Change the default value to false.
```C
bool debug_enclave = true;
```

This issue needs to be fixed in server code as well.